website/content/docs/secrets/databases/index.mdx

```diff
@@ -138,7 +138,7 @@ and private key pair to authenticate.
 | [Cassandra](/docs/secrets/databases/cassandra)        | Yes                      | Yes           | Yes (1.6+)   | Yes (1.7+)             | password                  |
 | [Couchbase](/docs/secrets/databases/couchbase)        | Yes                      | Yes           | Yes          | Yes (1.7+)             | password                  |
 | [Elasticsearch](/docs/secrets/databases/elasticdb)    | Yes                      | Yes           | Yes (1.6+)   | Yes (1.8+)             | password                  |
-| [HanaDB](/docs/secrets/databases/hanadb)              | Yes (1.6+)               | Yes           | Yes (1.6+)   | No                     | password                  |
+| [HanaDB](/docs/secrets/databases/hanadb)              | Yes (1.6+)               | Yes           | Yes (1.6+)   | Yes (1.8+)             | password                  |
```

Comment:
Should this be 1.12+ for username customization?

Reply:
Ah yes, you are right. I will update it again right now. Thank you JM!